src/services/pcn-nat/test/test_tcp_snat.sh

```diff
@@ -16,9 +16,9 @@ source "${BASH_SOURCE%/*}/helpers.bash"
 
 function test_tcp {
     sudo ip netns exec ns2 netcat -l -w 5 $tcp_port&
-	sleep 2
+	sleep 5
```

Comment:
Would it be possible to define a constant that is valid for all the tests, so that we don't have to change it manually throughout the entire file?

Reply:
I think this should be done in another PR as this change should affect every service's tests.

Reply:
Good point. Leave this as is.